modules/custom/activity_creator/activity.page.inc

```diff
@@ -36,12 +36,12 @@ function template_preprocess_activity(array &$variables) {
   // Display activity created date in format 'time ago'.
   $created_time_ago = \Drupal::service('date.formatter')
     ->formatTimeDiffSince($activity->getCreatedTime(), ['granularity' => 1, 'return_as_object' => TRUE]);
-  $date = t('%time ago', ['%time' => $created_time_ago->getString()]);
-  if ($full_url == '') {
+  $date = t('@time ago', ['@time' => $created_time_ago->getString()]);
+  if (isset($variables['elements']['#view_mode']) && ($variables['elements']['#view_mode'] === 'notification_archive' || $variables['elements']['#view_mode'] === 'notification')) {
```

Comment:
@jaapjan this exception for notification and notification_archive is already code, nice.
But maybe we can keep it consistent though and change this if statement to:

```if (isset($variables['elements']['#view_mode']) && in_array($variables['content']['field_activity_output_text']['#view_mode'], [
  'notification',
  'notification_archive',
]));```